broker/src/main/java/org/apache/rocketmq/broker/topic/TopicConfigManager.java

```diff
@@ -221,39 +222,23 @@ public TopicConfig createTopicInSendMessageBackMethod(
         final int clientDefaultTopicQueueNums,
         final int perm,
         final int topicSysFlag) {
-        TopicConfig topicConfig = this.topicConfigTable.get(topic);
-        if (topicConfig != null)
-            return topicConfig;
-
-        boolean createNew = false;
-
-        try {
-            if (this.lockTopicConfigTable.tryLock(LOCK_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)) {
-                try {
-                    topicConfig = this.topicConfigTable.get(topic);
-                    if (topicConfig != null)
-                        return topicConfig;
-
-                    topicConfig = new TopicConfig(topic);
-                    topicConfig.setReadQueueNums(clientDefaultTopicQueueNums);
-                    topicConfig.setWriteQueueNums(clientDefaultTopicQueueNums);
-                    topicConfig.setPerm(perm);
-                    topicConfig.setTopicSysFlag(topicSysFlag);
-
-                    log.info("create new topic {}", topicConfig);
-                    this.topicConfigTable.put(topic, topicConfig);
-                    createNew = true;
-                    this.dataVersion.nextVersion();
-                    this.persist();
-                } finally {
-                    this.lockTopicConfigTable.unlock();
-                }
-            }
-        } catch (InterruptedException e) {
-            log.error("createTopicInSendMessageBackMethod exception", e);
-        }
-
-        if (createNew) {
+        // Internal class requires final variables
+        final AtomicBoolean createNew = new AtomicBoolean(false);
+        TopicConfig topicConfig = this.topicConfigTable.computeIfAbsent(topic, key -> {
```

Comment:
Avoid lock contention of `lockTopicConfigTable`.

Reply:
@ConeyLiu why you polish createTopicInSendMessageBackMethod and skip createTopicInSendMessageMethod?